apps/apps.h

```diff
@@ -515,9 +515,9 @@ int do_X509_REQ_sign(X509_REQ *x, EVP_PKEY *pkey, const EVP_MD *md,
                      STACK_OF(OPENSSL_STRING) *sigopts);
 int do_X509_CRL_sign(X509_CRL *x, EVP_PKEY *pkey, const EVP_MD *md,
                      STACK_OF(OPENSSL_STRING) *sigopts);
-# ifndef OPENSSL_NO_PSK
+
```

Comment:
No, TLS1.3 does not turn PSK on. OPENSSL_NO_PSK only applies to TLSv1.2 and does not impact the TLSv1.3 PSK implementation. In the apps there are a few options that are re-used between the TLSv1.2 and TLSv1.3 implementations which is why there are a few changes to the guards in this area.